fhir-persistence-schema/src/main/resources/postgres/delete_resource_parameters.sql

```diff
@@ -40,6 +40,8 @@ BEGIN
 	USING p_logical_resource_id;
 	EXECUTE 'DELETE FROM {{SCHEMA_NAME}}.' || p_resource_type || '_security            WHERE logical_resource_id = $1'
 	USING p_logical_resource_id;
+    EXECUTE 'DELETE FROM {{SCHEMA_NAME}}.' || p_resource_type || '_ref_values WHERE logical_resource_id = $1'
```

Comment:
interesting mix of tabs and spaces in this file

Reply:
Not sure where they came from...the editor is supposed to replace tabs with spaces...perhaps it's not working for sql files. I'll take a look.

Reply:
fixed